tui/installer/screens.py

```diff
@@ -229,7 +229,7 @@ def get_installation_type(answers):
     # RAID members, for filtering out from upgradable-products and
     # backups, and to decide whether to propose to activate existing RAID.
     raid_members = []
-    if "assemble-raid" not in answers:
+    if constants.HAS_RAID_ASSEMBLE and "assemble-raid" not in answers:
```

Comment:
The feature flag to enable from presenting the "assemble RAID" option also disables the safety measure of filtering out upgrade/restore entries that the user **should not** select, as they indeed live on a RAID volume.  It is I think a bad idea to allow the user to do this, but then hiding those options without letting the user assemble RAID volumes looks like a bad idea too.  I would rather just let the behavior added by this PR be the default, and just not add this feature flag.

Reply:
As XenServer doesn't support installing on software RAID at all, it is better to keep the feature flag to hide the "assemble RAID" option. The button would be too confusing for users in a XenServer context.